### ISSUE TITLE
Bug: Searching for "Alabama A & M University" comes up with an error

### ISSUE THREAD

**Issue by saintsoup52** [18F/college-choice#1344]
Using the name search field... "Alabama A & M University" yields 0 results.

I am assuming it is because of the special character "&" in the school's name.

If I search of Alabama then the school comes up.

**Comment by jasonphillips** [comment]
This issue is not caused by the front end, but is a natural consequence of the Data API (open data maker), which uses the `autocomplete` data type for searching the school name, as documented here:

https://github.com/18F/open-data-maker/blob/api-docs/API.md#word-and-substring-matches-on-autocomplete-fields

```
Note that all given words have to be at least three characters long
```

The "A & M" of the university cannot be searched since the indexer breaks up your search into separate words and looks for matching words of at least length three.  

That being said, all the official information on Alabama A&M (http://www.aamu.edu/) writes its name without those spaces, as "Alabama A&M".  It seems that the underlying data itself should perhaps be updated to remove the spaces from the institution's name, and then it may be more searchable. I dont' know off the top of my head, however, whether the "&" will cause elasticsearch to tokenize that word incorrectly.

**Comment by saintsoup52** [comment]
Thank you for following up with this…

The Department of ED has a pretty strict naming convention for schools in IPEDS… hence why “A & M” in the data has spaces even though on their website logo does not.

Any help resolving this issue would be appreciated. ☺

**Comment by jasonphillips** [comment]
Good point, I wasn't thinking about the IPEDS naming restrictions that require the data to look this way.

It seems to me that the best option for improving school-name searches would be to add additional alternative titles or designations to the search indexer (ie., "Texas A & M University" should have an extra, hidden field containing "A&M" and "TAMU"). These alternative, hidden keywords could then be additionally searched when using the "title" field, and more universities would come up correctly on the kinds of common name searches that users expect when referring to higher ed institutions. 

IPEDS does have an institution "alias" field (IALIAS) that contains exactly that, with indeed those specific entries for Texas A&M ("Texas A&M | TAMU"), and the field was presumably built for exactly this purpose of better searching. Alabama A & M only has the alpha code there (AAMU), but presumably that is at least under institution control in their next routine upload of data to IPEDS. It would be great to see that field used, which would require updates to the data dictionary that is used to import the IPEDS data, as well as changes to the way the API is called to search that field.

**Comment by LisaGee** [comment]
Probably same fix as #1356

**Comment by barkimedes** [comment]
We also need to make sure that when this issue gets fixed, it is also fixed for Texas A&M University. This issue is referenced in an email thread related to that.

**Comment by barkimedes** [comment]
From another email thread. We need to make sure this is fixed for University of La Verne, too.

**Comment by hollyallen** [comment]
Also related to #187
